prow/cluster/jobs/istio-private/istio.io/istio-private.istio.io.master.gen.yaml

```diff
@@ -80,14 +80,124 @@ postsubmits:
     clone_uri: git@github.com:istio-private/istio.io.git
     cluster: private
     decorate: true
-    name: k8s-tests_istio.io_postsubmit_priv
+    name: k8s-tests-default_istio.io_postsubmit_priv
```

Comment:
can we remove the `k8s` prefixes throughout? It's not really helpful ... everything is k8s.